---

/**:
  ros__parameters:
    map_osm_file: "/home/drivingsim/workspace/dfg-car/src/cr2autoware/data/autonomoustuff_parking_lot.osm"
    latitude: 37.380811523812845
    longitude: -121.90840595108715
    elevation: 16.0
    origin_offset_lat: -5.239983224214484e-06
    origin_offset_lon: 4.5845488187978845e-06
    left_driving: False
    adjacencies: False 